src/evergreen.ts

```diff
@@ -21,7 +21,7 @@ export class client {
      * @returns nothing
      */
     public getResource(callback: request.RequestCallback, resource: string, params?: object) {
-        const url = this.serverURL + "/" + resource + queryString(params);
+        const url = resource + queryString(params);
```

Comment:
What was the reason for changing this? The base server URL needs to be there to send the request

Reply:
We changed this for testing purposes when I was checking to see if the models I built for the patches actually worked. I will change it back and try running the tests again.